demos/notebooks/demo_pipeline.ipynb

```diff
@@ -453,16 +683,54 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#%% PLOT COMPONENTS\n",
-    "cnm2.estimates.plot_contours_nb(img=Cn, idx=cnm2.estimates.idx_components)"
+    "cnmf_model = cnmf.CNMF(n_processes, \n",
+    "                       params=parameters, \n",
+    "                       dview=multiprocessing_pool)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once initialized, we we could immediately run `cnmf_model.fit(images)` if we knew the parameters were good -- the parameters in your CNMF estimator object are accessable in `cnmf_model.params`. However, before running the algorithm, let's discuss the more critical parameters that you will most likely need to tweak when running CNMF on your own data. "
```

Comment:
Does all this really belong in the notebook? Why not point to common docs? This adds a lot of bulk to the notebook (that might be duplicated to most notebooks if you want to be consistent).